definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-bailiff-nonprod.json

```diff
@@ -1,4 +1,130 @@
 [
+  {
+    "LiveFrom": "01/02/2021",
```

Comment:
`LiveFrom` is ignored y CCD, but can you please make sure it has valid creation date of a definition?